.travis.yml

```diff
@@ -12,7 +12,7 @@ jdk:
     - oraclejdk8
     
 script:
-    - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent package -P sonar sonar:sonar -B
+    - mvn clean verify-P sonar -B
```

Comment:
This must be a mistake - I think there's a space missing here. Also, I tried it locally and I couldn't see it uploading to Sonar still.

Reply:
yeah i pushed it from mobile damn xD